server.go

```diff
@@ -1638,11 +1642,11 @@ out:
 		case qmsg := <-s.query:
 			s.handleQuery(state, qmsg)
 
-		// Shutdown the peer handler.
+		// Disconnect the peer handler.
 		case <-s.quit:
-			// Shutdown peers.
+			// Disconnect peers.
 			state.forAllPeers(func(sp *serverPeer) {
-				sp.Shutdown()
```

Comment:
Since the `Shutdown` function was removed an it has the trace logging for the peer being forcefully shutdown, I'd suggest adding the removed logging statement back to here before calling `Disconnect`.